modules/serverpod_auth/serverpod_auth_server/lib/src/endpoints/status_endpoint.dart

```diff
@@ -16,11 +16,24 @@ class StatusEndpoint extends Endpoint {
     return userId != null;
   }
 
-  /// Signs out a user.
-  Future<void> signOut(Session session) async {
+  /// **[Deprecated]** Signs out a user from all sessions.
+  /// Use `signOutCurrentDevice` for the current session or `signOutAllDevices` for all sessions.
+  @Deprecated(
+      'Use signOutCurrentDevice for the current session or signOutAllDevices for all sessions. This method will be removed in future releases.')
+  Future<void> signOutUser(Session session) async {
     await UserAuthentication.signOutUser(session);
   }
```

Comment:
Suggestion: Revert this change.

This is a breaking change and should not be part of this PR.
This would break all old clients.

Do we need a test that calls the old endpoint to validate we don't introduce this change?

Reply:
Reverted name change. 
The test is required to ensure the method's behavior remains unchanged with no functional modifications.